README.md

```diff
@@ -1,56 +1,55 @@
-# Secret-SCRT - Privacy coin backed by SCRT
+# SNIP-20 Reference Implementation
 
-This is a PoC of how to create a privacy token on the Secret Network backed by a native coin.
+This is an implementation of a SNIP-20 compliant token contract.  At the time of token creation you may configure:
+* Public Total Supply:  If you enable this, the token's total supply will be displayed whenever a TokenInfo query is performed.  DEFAULT: false
+* Enable Deposit: If you enable this, you will be able to convert from SCRT to the token.*  DEFAULT: false
+* Enable Redeem: If you enable this, you will be able to redeem your token for SCRT.*  It should be noted that if you have redeem enabled, but deposit disabled, all redeem attempts will fail unless someone has sent SCRT to the token contract.  DEFAULT: false
+* Enable Mint: If you enable this, any address in the list of minters will be able to mint new tokens.  The admin address is the default minter, but can use the set/add/remove_minters functions to change the list of approved minting addresses.  DEFAULT: false
+* Enable Burn: If you enable this, addresses will be able to burn tokens.  DEFAULT: false
 
-Usage is pretty simple - you deposit SCRT into the contract, and you get SSCRT (or Secret-SCRT), which you can then use with the ERC-20-like functionality that the contract provides including: sending/receiving/allowance and withdrawing back to SCRT. 
 
-In terms of privacy the deposit & withdrawals are public, as they are transactions on-chain. The rest of the functionality is private (so no one can see if you send SSCRT and to whom, and receiving SSCRT is also hidden). 
-
-The code was updated with a new mechanism, which I call viewing keys. This allows a user to generate a key that enables off-chain queries. This way you can perform balance and transaction history queries without waiting for a transaction on-chain. The tranaction to create a viewing key is expensive, to the tune of about 3M gas. This is intended to make queries take a long time to execute to be resistant to brute-force attacks.
-
-The usual disclaimer: Don't use this in production, I take no responsibility for anything anywhere anytime etc.
+\*:The conversion rate will be 1 uscrt for 1 minimum denomination of the token.  This means that if your token has 6 decimal places, it will convert 1:1 with SCRT.  If your token has 10 decimal places, it will have an exchange rate of 10000 SCRT for 1 token.  If your token has 3 decimal places, it will have an exchange rate of 1000 tokens for 1 SCRT.  You can use the exchange_rate query to view the exchange rate for the token.  The query response will display either how many tokens are worth 1 SCRT, or how many SCRT are worth 1 token.  That is, the response lists the symbol of the coin that has less value (either SCRT or the token), and the number of those coins that are worth 1 of the other.
```

Comment:
Why not let the creator set the conversion rate?

Reply:
The conversion rate applies to tokens like sSCRT. The conversion rate is between the wrapped asset and the corresponding native asset, so unless you're doing something special, this is in line with how `Withdraw` and `Redeem` are defined.